vingo/main.go

```diff
@@ -27,11 +31,15 @@ func main() {
 
 	api := fiber.New(fiber.Config{})
 
-	api.Use(cors.New(cors.Config{
-		AllowOrigins:     corsAllowOrigins,
-		AllowHeaders:     "Origin, Content-Type, Accept, Access-Control-Allow-Origin",
-		AllowCredentials: true,
-	}))
+	if development {
+		api.Use(cors.New(cors.Config{
+			AllowOrigins:     corsAllowOrigins,
+			AllowHeaders:     "Origin, Content-Type, Accept, Access-Control-Allow-Origin",
+			AllowCredentials: true,
+		}))
+	} else {
+		api.Static("/", "./public")
+	}
```

Comment:
this works in the production container, until you refresh, at which point it will request eg /scans from the server and it will serve the json response from the backend instead.

Possible fix: add a catchall route so all unknown paths route to the frontend path
This means that the backend functions should be behind a /api path again, so they don't interfere, since we have some double routes (/login, /cards, /scans, ...)

Reply:
Fixed in [1d16733](https://github.com/ZeusWPI/ZeSS/pull/46/commits/1d1673341e404c69f890fd673c7d045834c96bc9)

I've opted to do just that.
Move the backend paths to behind /api en add a catch all serving the frontend